zip-0202.rst

```diff
@@ -192,7 +192,7 @@ Tests can continue to check for the existence of forwards-compatible transaction
         }
     }
 
-When (de)serializing v3 transactions, the version group id should also be checked in case the transaction is intended for a branch which has a different format for its version 3 transaction::
+When (de)serializing v3 transactions, the version group id must also be checked in case the transaction is intended for a branch which has a different format for its version 3 transaction::
```

Comment:
Note: this is not "MUST" because it's referring to the C++ implementation, which is not part of the spec.